schema.sql

```diff
@@ -62,3 +62,7 @@ CREATE TABLE visits (
     animals_id BIGINT REFERENCES animals (id),
     date_of_visit DATE NOT NULL
 );
+
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
```

Comment:
* Kindly add indexes for FOREIGN KEYS or on columns frequently used in queries, this is one of the basic solutions to performance problems ( ie `CREATE INDEX index_name
ON table_name (column1, column2, ...);` ). Kindly apply this and see improvement in execution time 🙏 [Read More here](https://www.w3schools.com/sql/sql_create_index.asp)

Reply:
@Davidosky007  Thanks for the notice. 😄